src/printer.ts

```diff
@@ -98,7 +98,8 @@ export interface PugPrinterOptions {
 export class PugPrinter {
 	private result: string = '';
 
-	private currentIndex: number = 0;
+	// The current index in the `tokens` array, starts at -1, because `getNextToken()` increases before retrieving
```

Comment:
Could you convert this to JSDoc?
So on hover we can see the comment.
Don't know if we should split the descriptive part from the developer comment part :thinking: